src/app/pages/profile/profile/profile.component.ts

```diff
@@ -111,7 +111,7 @@ export class ProfileComponent extends PageComponent {
     }
 
     public openStatsPopup(set: GearSet): void {
-        this.dialog.open(StatsEditPopupComponent, {data: set});
+        this.dialog.open(StatsEditPopupComponent, {data: {set: set, jobs: this.craftingJobs.slice(0, 8)}});
```

Comment:
Why do you slice `this.craftingJobs` here? is it to pass a copy?

Reply:
`craftingJobs` is actually a misnomer because the last 3 objects are DoL jobs. The slice here ensures we are only passing in DoH jobs.